draft-ietf-dnsop-structured-dns-error.md

```diff
@@ -634,7 +634,7 @@ registry {{IANA-DNS}}:
 
 # Interoperation with RPZ Servers
 
-This appendix discusses operation with an Response Policy Zones (RPZ) server {{RPZ}} that
+This appendix provides a non-normative guidance for operation with an Response Policy Zones (RPZ) server {{RPZ}} that
```

Comment:
I understand IETF has not defined RPZ but it is a popular deployment. If the guidance given in this section is not followed, it will break interoperation with RPZ.

Reply:
True, but still the text is informative. Otherwise, we will need to be more precise and use the normative language for the behavior.

I can change "provides a non-normative guidance for"  to "provides a non-normative guidance for... This guidance is provide to ease interoperation with RPZ."

Reply:
Works for me. Thanks.